plaid/payment.go

```diff
@@ -145,6 +152,7 @@ type createStandingOrderRequest struct {
 	Reference   string           `json:"reference"`
 	Amount      PaymentAmount    `json:"amount"`
 	Schedule    *PaymentSchedule `json:"schedule"`
+	Options     *PaymentOptions  `json:"options,omitempty"`
```

Comment:
I realize this is a remnant of a prior PR, but is there a reason `createStandingOrderRequest` needs to exist separately from `createPaymentRequest`?

It seems like the only difference is the `schedule` field, which could be marked with `omitempty`? That would simplify all of the conditionals below